File: TCDNew-master/src/ui.android/lib/models/dtos/PurchaseTaskCreateFromPacsReq.dart
import 'PacKeyDto.dart';

class PurchaseTaskCreateFromPacsReq {
  String facilityId;
  List<PacKeyDto> pacs;

  PurchaseTaskCreateFromPacsReq({this.facilityId, this.pacs});

  toJson() {
    return {
      'facilityId': facilityId,
      'pacs': pacs,
    };
  }
}

File: TCDNew-master/src/api/Cen.Wms.Domain.Purchase.Api/Dtos/PurchaseTaskCreateFromPacsReq.cs
using System;
using System.Collections.Generic;

namespace Cen.Wms.Domain.Purchase.Api.Dtos
{
    public class PurchaseTaskCreateFromPacsReq
    {
        public Guid FacilityId { get; set; }
        public IEnumerable<PacKeyDto>  Pacs { get; set; }
    }
}